jest.config.js

```diff
@@ -192,4 +192,7 @@ module.exports = {
 
   // Whether to use watchman for file crawling
   // watchman: true,
+  moduleNameMapper: {
```

Comment:
I don't think we need this. I already installed  `identity-obj-proxy` and added it to this file under `moduleNameMapper`.  

 
- [__mocks__](https://github.com/sensational-sycamore/sycamore/tree/main/__mocks__)
- [moduleNameMapper]( https://github.com/sensational-sycamore/sycamore/blob/b7e3e6410bb40b1dfb2477d41dddba888e07578c/jest.config.js#L92)

Please run `git pull origin main` in your repository to get the latest from main in it. You might need to fix conflicts, but better fix is in you local. 

Look at `Handling Static Assets` vs `Mocking CSS Modules` in here: https://jestjs.io/docs/webpack